bin/docker_start.sh

```diff
@@ -15,12 +10,8 @@ uwsgi_threads=${UWSGI_THREADS:-1}
 
 mountpoint=${SUBPATH:-/}
 
-until pg_isready; do
-  >&2 echo "Waiting for database connection..."
-  sleep 1
-done
-
->&2 echo "Database is up."
+# wait for required services
+/wait_for_db.sh
```

Comment:
```suggestion
./wait_for_db.sh
```
without the `.` I can't run this script locally anymore outside of a docker context (and that is something I frequently have to do when debugging uwsgi stuff :wink: )